### PR TITLE
Feature/revise dockerfile

### PR DESCRIPTION
**Pull request by rulixxx**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,4 +1,4 @@
-FROM  quay.io/wtsicgp/dockstore-cgpmap:3.1.4 as builder
+FROM quay.io/wtsicgp/pcap-core:5.2.1 as builder
```
Comment (keiranmraine) [inline Dockerfile:1]: may as well use 5.2.2

File: setup.sh [hunk 2]
```diff
@@ -85,80 +66,31 @@ PERLROOT=$INST_PATH/lib/perl5
...
-# cleanup all junk
-rm -rf $SETUP_DIR
+bash build/opt-build.sh $INST_PATH
+bash build/opt-build-local.sh $INST_PATH
 
 echo
 echo
 echo "Please add the following to beginning of path:"
 echo "  $INST_PATH/bin"
-echo "Please add the following to beginning of PERL5LIB:"
```
Comment (keiranmraine) [inline setup.sh:160]: Users of the setup.sh route will still need to add to the perl5lib

File: Dockerfile (hunk 1 above)
Comment (keiranmraine) [reply Dockerfile:1]: this is not essential

Comment (rulixxx) [reply Dockerfile]: Done

File: setup.sh (hunk 2 above)
Comment (rulixxx) [reply setup.sh]: Done

**Pull request merged**